Makefile

```diff
@@ -61,10 +60,6 @@ api_docs:
 	@echo "Installing swagger-merger" && npm install swagger-merger -g
 	@swagger-merger -i $(PWD)/swagger/index.yaml -o $(PWD)/docs/api_docs/bundle.yaml
 
-checks:
-	@echo "Check deps"
-	@(env bash $(PWD)/buildscripts/checkdeps.sh)
```

Comment:
no need to check deps or in GOPATH, flagr is fully on go.mod and each build/run go command comes with `GO111MODULE=on`